geometry3Sharp_netstandard.csproj

```diff
@@ -10,8 +10,11 @@
     <PackageLicenseUrl>https://github.com/gradientspace/geometry3Sharp/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/gradientspace/geometry3Sharp</PackageProjectUrl>
     <PackageTags>geometry3;graphics;math;approximation;solvers;color;convexhull;meshes;spatial;curves;solids;3d;unity</PackageTags>
-	<RepositoryUrl>https://github.com/gradientspace/geometry3Sharp</RepositoryUrl>
-	<RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/gradientspace/geometry3Sharp</RepositoryUrl>
```

Comment:
should not we have a link to our repository instead?

Reply:
Until it's our repository, but a fork -- no, we shouldn't